packages/project-utils/testing/presets/index.js

```diff
@@ -73,17 +73,17 @@ const getPackagesPresets = (targetKeywords, nameSuffix) => {
         return `${name}-${nameSuffix}`;
     };
     /**
-     * We go through all available packages to build presets for them.
+     * We go through all available packages to build ToolbarPresets for them.
      */
     for (const pkg of packages) {
         const name = getPackageName(pkg);
-        const presetsPath = path.join(pkg.path, "__tests__/__api__/presets.js");
+        const presetsPath = path.join(pkg.path, "__tests__/__api__/ToolbarPresets.js");
```

Comment:
@Pavel910 I think that this needs to be fixed as well

Reply:
@mihajlovco yes, good catch